src/main/kotlin/org/akuleshov7/utils/Credentials.kt

```diff
@@ -0,0 +1,20 @@
+package org.akuleshov7.utils
+
+@Suppress("USE_DATA_CLASS")
+class Credentials(auth: String) {
+    val gitHubUserName: String
+    val gitHubAuthToken: String
+
+    init {
+        auth.let {
+            val splitAuth = auth.split(":")
+            if (splitAuth.size != 2) {
+                """Incorrect value ($auth) is passed to 'auth' property.
```

Comment:
I don't think you should log it here - user sees the command they just entered, and logging secret token might be considered a security risk.

Reply:
Completely agree